### ISSUE TITLE
How to GraphQL's future

### ISSUE THREAD

**Issue by itscarlosrufo** [howtographql/howtographql#896]
Hi everyone,

During these days some of the contributors have been working on fixing all the issues related to the existing tutorials. They're already back & working as the docs define!

I'd like to appreciate all the work done by the community creating this incredible teaching platform giving the opportunity to people on getting introduced to GraphQL.

As 'official' GraphQL learning platform as it's being linked from [graphql.org](https://graphql.org/learn/) and [graphql.com](https://www.graphql.com/tutorials/) , it should ensure:

- The tutorials are up-to-date, as well as frequently maintained
- Include a variety of languages/implementations.
- Promote inclusivity, community & transparency.

Being honest, it is being difficult to achieve. Although all collaborators are working hard on it, it's far away from what it should be.

As the GraphQL community grows, there're a lot of people willing to learn/contribute. IMO we cannot stop them from improving the existing resources & create new ones, else we should provide guidance and give them the tools to do so.

I know that this requires a lot of time & effort but I strongly believe that collaborating all together we could easily have a platform where everyone could learn about everything.

I'm gonna ping some of the GraphQL community leaders with the aim of start a discussion and hopefully find a solution to this problem.

@leebyron @schrockn @schickling @martijnwalraven @coco98 @KyleAMathews @xuorig @dabit3 @eveporcello @Urigo @ladyleet @stubailo

Thanks to all contributors for make this platform happen!

**Comment by wtrocki** [comment]
Ad1. Tracking releases of GraphQL spec/GraphQL libraries can help to address some issues and migrate tutorials.

Ad 2. Setting up something like http://zanata.org and adding "Translate this to your native language" button on the website could be the solution for that. Many people could help with translations while learning GraphQL themselves.  

Ad 3. This could be really good topic for graphql foundation that is currently forming.
See: https://groups.google.com/a/linuxfoundation.org/forum/#!forum/graphql-formation
Link to website: https://gql.foundation

**Comment by jonatasbaldin** [comment]
Yeah, totally agree! My points for the future of the project are:

- Be open to get more contributors/maintainers and a have process to onboard them
- Invite and help from the community to create more diverse tutorials
- Have, with the community, a roadmap for the project
- Have an open discussion platform (e.g. Slack)

And also, on the GraphQL Foundation side, would be nice to have project under their umbrella, since it's one of the best resources to learn GraphQL today.

**Comment by jonatasbaldin** [comment]
Oops, I didn't know I had unpinned this hahaha pinned again!

**Comment by nazarepiedady** [comment]
@itscarlosrufo, @jonatasbaldin, @wtrocki what do you think about reviving this issue?
